model/src/main/kotlin/ScanSummary.kt

```diff
@@ -76,4 +76,6 @@ data class ScanSummary(
 ) {
     @get:JsonIgnore
     val licenses: Set<SpdxExpression> = licenseFindings.mapTo(mutableSetOf()) { it.license }
+
+    var scannerSpecificResults = mutableMapOf<String, Any>()
```

Comment:
What kind of information would be stored in this property? Couldn't this be part of `ScanResult.rawResult`?

Reply:
I tried to stick to the ScanSummary model as much as possible : for instance I am filling found copyrights and found licences.

However the idea was that the scanner does some transformation of the raw result to make them more useful.
@sschuberth requested for instance the following logic : if some files have been manually marked as identified in the UI, they are the result. If there are none, the files identified by FossId itself are the result.
You can see the logic in FossId#228

Since there was no right place in ScanSummary to put them, and since these are no raw results because they have been transformed, I thought this extension 'scannerSpecificResults' would do the trick.

Reply:
My concerns about this are a bit similar to the concerns about the changes in `LicenseFinding`. Is this property just added for reference, or will it be used by ORT for something later on? Also, adding it here means it's not respected by the generated `copy` function which is for example used in `SubtractScanResultsCommand`.
As it seems this property is not mandatory for the remote scanner to work, I would suggest to move it to a separate PR as well, to minimize the changes to the data model in this PR.

Reply:
Agreed.

Reply:
I removed it and will do a separate PR.

Same comment as above : I am processing FossID results in the scanner and I found a pity that only some of the findings can be carried to the ScanSummary (License and Copyright).
Since I am doing the effort to process the result, why is it the job of someone else to do it again and process the raw results ?
But this discussion will be the topic of the other PR.

done.